src/librbd/cache/pwl/ImageCacheState.cc

```diff
@@ -141,19 +143,10 @@ ImageCacheState<I>* ImageCacheState<I>::create_image_cache_state(
     }
 
     bool cache_exists = (bool)f["present"];
-    int cache_type = (int)f["cache_type"];
-
-    switch (cache_type) {
-      case IMAGE_CACHE_TYPE_SSD:
```

Comment:
Hi @MahatiC ，here I have a question. Why didn't here create a `cache_state` when the type was `IMAGE_CACHE_TYPE_SSD`, was this a error before?

Reply:
@CongMinYin 
```
case IMAGE_CACHE_TYPE_SSD:
case IMAGE_CACHETYPE_RWL:
// create cache_state
```
That syntax means, for both cases, create the `cache_state`. You can refer to the syntax details here[0]

[0]https://www.learncpp.com/cpp-tutorial/switch-fallthrough-and-scoping/

Reply:
> @CongMinYin
> 
> ```
> case IMAGE_CACHE_TYPE_SSD:
> case IMAGE_CACHETYPE_RWL:
> // create cache_state
> ```
> 
> That syntax means, for both cases, create the `cache_state`. You can refer to the syntax details here[0]
> 
> [0]https://www.learncpp.com/cpp-tutorial/switch-fallthrough-and-scoping/

Sorry for my misunderstanding.